hermes_eea/calibration/calibration.py

```diff
@@ -36,33 +33,37 @@
 def process_file(data_filename: Path) -> list:
     """
     This is the entry point for the pipeline processing.
-    It runs all of the various processing steps required
-    to create a L1A Hermes CDF File
+    It runs all of the various processing steps required to create a L1A Hermes CDF File.
     calls:
-        calibrate_file()
-           parse_science
-           CCSDSPY (parse_L0_sci_packets())
-           l0_sci_data_to_cdf()
-                SkymapFactory()
-                Use HermesData to populate CDF output file
-                Write the File
-        A Custom EEA SkymapFactory
-        HermesData
+
+    .. code-block:: python
+
+        calibrate_file(...)
+            parse_science_filename(...)
+            parse_l0_sci_packets(...)
+            l0_sci_data_to_cdf(...)
+                SkymapFactory(...)
+                # Use HermesData to populate CDF output file
+                Hermes_EEA_Data_Processor(...).build_HermesData()
+                # Write the File
+                hermes_eea_data.save(...)
+        # A Custom EEA SkymapFactory
+        # HermesData
+
 
     Parameters
     ----------
-    data_filename: str
+    data_filename: `pathlib.Path`
         Fully specificied filename of an input file.
-        The file contents:
-            Traditional binary packets in CCSDS format
+        The file contents: Traditional binary packets in CCSDS format
 
 
     Returns
     -------
-    output_filenames: list
-        Fully specificied filenames for the output files.
-    The file contents:
-        CDF formatted file with n packets iincluding time and [41,4,32] skymap.
+    output_filenames: `list[pathlib.Path]`
```

Comment:
```suggestion
    output_files: `list[pathlib.Path]`
```
Just to match variable name below